.github/workflows/bash-scripts/reservationcharges.sh

```diff
@@ -6,29 +6,24 @@ IFS="|"
 data_source="reservation_charges"
 
 # source file specific vars
+date_range_start=$(date -d '1 month ago' +"%Y-%m-01")
```

Comment:
this date command doesn't work on a mac, to make it easier to test for people it would be better to detect OS  and use gdate instead on mac

https://github.com/hmcts/dtspo-daily-monitoring/blob/20346804a7ce43eab74ac44f235f72dc040b2ec2/scripts/cert-check.sh#L8C1-L12

Reply:
added this check